test/e2e/network/service.go

```diff
@@ -3731,6 +3731,78 @@ var _ = common.SIGDescribe("Services", func() {
 
 		framework.Logf("Collection of services has been deleted")
 	})
+	/*
+		Release: v1.26
+		Testname: Service, same ports with different protocols on a Load Balancer Service
+		Description: Create a LoadBalancer service with two ports that have the same value but use different protocols. Add a Pod that listens on both ports. The Pod must be reachable via the ClusterIP and both ports
+	*/
+	ginkgo.It("should serve endpoints on same port and different protocol for internal traffic on Type LoadBalancer ", func() {
```

Comment:
Once upon a time we tried to flatten all LB tests into one run (because LB creation can be very slow).  I sort of lost touch - is that still something we care about?

@aojea

Reply:
we are not waiting for the LoadBalancer IP assignment in this test, we only care about the ClusterIP here